src/Bindings/PluginManager.cpp

```diff
@@ -961,14 +999,14 @@ bool cPluginManager::CallHookPlayerJoined(cPlayer & a_Player)
 
 
 
-bool cPluginManager::CallHookPlayerLeftClick(cPlayer & a_Player, int a_BlockX, int a_BlockY, int a_BlockZ, char a_BlockFace, char a_Status)
+bool cPluginManager::CallHookPlayerOpeningWindow(cPlayer & a_Player, cWindow & a_Window)
```

Comment:
Why the move? The functions were kinda alpha-sorted before, now it doesn't make sense

Reply:
I moved the two functions, because I had old hook names. Not sure why the change looks that strange ...

Reply:
Just swap the functions back into the order they were in.